graph/common/Encoding.java

```diff
@@ -52,6 +54,30 @@
     public static final String ROCKS_SCHEMA = "schema";
     public static final int ENCODING_VERSION = 1;
 
+    public enum Partition {
+        DEFAULT((short) 0, null),
+        VARIABLE_START_EDGE((short) 1, ByteArray.encodeString("VARIABLE_START_EDGE", STRING_ENCODING)),
```

Comment:
Define the encoding for the other partitions, with the appropriate ID and partition name. A partition name is a String that is converted into `ByteArray` using `STRING_ENCODING`.